ui/shared/BlockContactConfirmationDialog.qml

```diff
@@ -20,6 +20,7 @@ ModalPopup {
         anchors.left: parent.left
         anchors.right: parent.right
         wrapMode: Text.WordWrap
+        color: Style.current.textColor
```

Comment:
No need to do this. Just change `Text` to `StyledText`. It handles the color and the font